book.rb

```diff
@@ -0,0 +1,15 @@
+# This book class shows a Book as being an object in the Library(it hasa  title & author attribue & a rentals Array)
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Getters and Setters for @rentals should be present

` attr_accessor :rentals`